app/services/sign_in/logout_redirect_generator.rb

```diff
@@ -4,10 +4,10 @@
 
 module SignIn
   class LogoutRedirectGenerator
-    attr_reader :user, :client_config
+    attr_reader :user_verification, :client_config
 
-    def initialize(user:, client_config:)
-      @user = user
+    def initialize(user_verification:, client_config:)
```

Comment:
🗣️ Does it make sense to pass in the `credential_type` directly instead of `user_verification`? It looks like we only really need the credential type.